config/jetty.yml

```diff
@@ -1,10 +1,10 @@
 development:
   startup_wait: 15
-  jetty_port: 8983
+  jetty_port: 8980
```

Comment:
It doesn't have to be in this PR -- I created #43 for upgrading to solr_wrapper.